src/encoded/schemas/ivp_a1.json

```diff
@@ -0,0 +1,682 @@
+      "type": "string",
+      "enum": [
+        "1 ADC advertisement (e.g., website, mailing, newspaper ad, community presentation)",
+        "2 News article or TV program mentioning the ADC study",
+        "3 Conference or community event (e.g., community memory walk)",
+        "4 Another organization's media appeal or website (e.g., Alzheimer's Association, clinicaltrials.gov)",
+        "8 Other",
+        "9 Unknown"
+      ]
+    },
+    "prestat": {
+      "title": "3. Presumed disease status at enrollment:",
+      "type": "string",
+      "enum": [
+        "1 Case, patient, or proband",
+        "2 Control or normal",
+        "3 No presumed disease status"
+      ]
+    },
+    "prespart": {
+      "title": "4. Presumed participation:",
+      "type": "string",
+      "enum": [
+        "1 Initial evaluation only",
+        "2 Longitudinal follow-up planned"
+      ]
+    },
+    "source": {
+      "title": "5. ADC enrollment type:",
+      "type": "string",
+      "enum": [
+        "1 Primarily ADC-funded (Clinical Core, Satellite Core, or other ADC Core or project)",
+        "2 Subject is supported primarily by a non-ADC study (e.g., RO1, including non-ADC grants supporting the FTLD Module participation)"
+      ]
+    },
+    "birthmo": {
+      "title": "6a. Subject's month of birth(MM)",
+      "description": "Date can be submitted as MM.",
+      "type": "integer",
+      "minimum": 1,
+      "maximum": 12
+    },
+    "birthyr": {
+      "title": "6b. Subject's year of birth(YYYY)",
+      "description": "Date can be submitted as YYYY.",
+      "type": "integer",
+      "minimum": 1900
+    },
+    "sex": {
+      "title": "7. Subject's sex:",
+      "type": "string",
+      "enum": [
+        "1 Male",
+        "2 Female"
+      ]
+    },
+    "hispanic": {
+      "title": "8. Does the subject report being of Hispanic/Latino ethnicity (i.e. having origins from a mainly Spanish-speaking Latin American country), regardless of race?",
+      "type": "string",
+      "enum": [
+        "0 No (If No, SKIP TO QUESTION 9)",
+        "1 Yes",
+        "9 Unknown (If Unknown, SKIP TO QUESTION 9)"
+      ]
+    },
+    "hispor": {
+      "title": "8a. If yes, what are the subject's reported origins?",
+      "description": "If Other(SPECIFY),CONTINUE TO 8a1",
+      "type": "string",
+      "enum": [
+        "1 Mexican, Chicano, or Mexican-American",
+        "2 Puerto Rican",
+        "3 Cuban",
+        "4 Dominican",
+        "5 Central American",
+        "6 South American",
+        "50 Other, (specify)",
+        "99 Unknown"
+      ]
+    },
+    "hisporx": {
+      "title": "8a1. Other (specify subjects reported origins):",
+      "description": "(Any text or numbers with the exception of single quotes ('), double quotes (\"), ampersands (&), and percentage signs (%).)",
+      "type": "string"
+    },
```

Comment:
![image](https://user-images.githubusercontent.com/38848214/83456809-05214380-a426-11ea-8e79-69bb07d2acf1.png
What kind of pattern we need to set ? Same as other 'x' field.